scripts/ci/run_perftest.sh

```diff
@@ -222,13 +223,13 @@ query_types() {
 # clear. This function will translate the aliased query use cases to their
 # dataset use cases. Effectively this means "for this query use case, run the
 # queries against this dataset use case".
-query_usecase_alias() {
+queries_for_dataset() {
   case $1 in
-    window-agg|group-agg|bare-agg|ungrouped-agg|group-window-transpose|iot|group-window-transpose-low-card)
```

Comment:
Note `group-window-transpose`, `cardinality` usecases that didn't actually ever get passed in, since those aren't real query usecases.